crates/sncast/src/main.rs

```diff
@@ -432,7 +432,7 @@ async fn run_async_command(
         }
         Commands::Verify(verify) => {
             let manifest_path = assert_manifest_path_exists()?;
-            let package_metadata = get_package_metadata(&manifest_path, &None)?;
+            let package_metadata = get_package_metadata(&manifest_path, &verify.package)?;
```

Comment:
It'd be better if `&package_metadata.manifest_path` is passed instead of `&manifest_path` in the `verify function call `, [ref](https://github.com/foundry-rs/starknet-foundry/blob/b634b7a1a93b2cdd4d3865330a889c1ad5b4a011/crates/sncast/src/main.rs#L451)